slackclient/slackrequest.py

```diff
@@ -1,28 +1,31 @@
-import requests
 import json
+import platform
+import requests
 import six
 import sys
-import platform
+
 from .version import __version__
 
 
 class SlackRequest(object):
-    def __init__(self, proxies=None):
-
-        # __name__ returns 'slackclient.slackrequest', we only want 'slackclient'
-        client_name = __name__.split('.')[0]
-        client_version = __version__  # Version is returned from version.py
+    def __init__(
+            self,
+            proxies=None,
+            domain="slack.com"
```

Comment:
i think removing `domain` as an option from `do()` and adding it as a option for the constructor would be unrelated and a breaking change. can we revert that part?